### PR TITLE
feat(RHIF-22): enable AAP & MSSQL fitlers

### PR DESCRIPTION
**Pull request by mkholjuraev**
resolves: https://issues.redhat.com/browse/RHIF-22

### CODE REVIEW

File: src/Utilities/Helpers.js [hunk 1]
```diff
@@ -244,6 +245,16 @@ export const getFilterValue = (category, key) => {
 };
 
 export const encodeParams = (parameters, shouldTranslateKeys) => {
+    const calculateWorkloads = (systemProfile) => {
+        let result = '';
+        Object.entries(generateFilter({ system_profile: systemProfile })).forEach(entry => {
+            const [key, value] = entry;
+            result = result + '&' + `${key}=${value}`;
```
Comment (mkholjuraev) [inline src/Utilities/Helpers.js:252]: I will update string concatenating on line 252

**Comment by mkholjuraev** [comment]
@RedHatInsights/team-interact this PR is ready to be reviewed if you would like to have a look.

**Review by johnsonm325 — APPROVED** [review]
It all looks complex to me. haha. If I'm correct, you use `mapGlobalFilters` to build a data structure you can use throughout the app. Then then functions in `encodeParams` are used to build the url for hitting the api.

If I'm correct, then it seems to work properly. Nice job.

I think this is something that can be used across apps. Maybe we eventually pull this out into frontend-components.

**Comment by mkholjuraev** [comment]
@johnsonm325 thanks for the review.  

Yes, you are right. This was what I wanted to say when I talked about **mapGlobalFilters** in the team-interact channel.

**Pull request merged**